app/src/main/java/com/orange/ods/app/ui/utilities/composable/CodeImplementation.kt

```diff
@@ -23,88 +22,160 @@ import androidx.compose.ui.semantics.semantics
 import com.orange.ods.app.R
 import com.orange.ods.compose.theme.OdsTheme
 
+const val IconPainterValue = "<icon painter>"
+const val ImagePainterValue = "<image painter>"
+const val CardTextValue = "<card text>"
+
+abstract class CodeParameter(val name: String) {
+    abstract val code: @Composable () -> Unit
+}
+
+open class StringParameter(name: String, val value: String) : CodeParameter(name) {
+    override val code
+        get() = @Composable {
+            TechnicalText(text = "$name = $value,")
+        }
+}
+
+open class StringRepresentationParameter<T>(name: String, value: T) : StringParameter(name, value.toString())
+open class BetweenQuotesStringParameter(name: String, textValue: String) : StringParameter(name, "\"$textValue\"")
```

Comment:
Yes, I agree. I did a lot of renaming and I had a `SimpleParameter` at a time but I was not really satisfied. I refactored many things since and I think it's better to adopt this wording now.